.github/workflows/android.yml

```diff
@@ -28,6 +28,11 @@ jobs:
         if: ${{ matrix.platform.name == 'Android.mk' }}
         run: |
           ./build-scripts/androidbuildlibs.sh
+      - uses: actions/setup-java@v3
+        if: ${{ matrix.platform.name == 'CMake' }}
+        with:
+          distribution: 'temurin'
+          java-version: '8'
```

Comment:
I noticed [this recent commit](https://github.com/libsdl-org/SDL/commit/b701ac026661d2a5be0079ed023bb75d4f651e26) for Java 1.7 support. 
8 is the oldest Java release that still has support (LTS).

Reply:
Looks like this fails to create a source jar archive so I bumped it to 11.
log: https://github.com/libsdl-org/SDL/actions/runs/4853935644/jobs/8650710471

Reply:
I fixed the original error, but still have to bump the java version because of the following error when running `d8`
```
cd /home/runner/work/SDL/SDL/build/test && /usr/local/lib/android/sdk/build-tools/34.0.0/d8 /home/runner/work/SDL/SDL/build/test/checkkeysthreads.jar /home/runner/work/SDL/SDL/build/SDL3-3.0.0.jar --lib /usr/local/lib/android/sdk/platforms/android-34/android.jar --output /home/runner/work/SDL/SDL/build/test/CMakeFiles/checkkeysthreads-dex.dir
Error: A JNI error has occurred, please check your installation and try again
Error: Exception in thread "main" java.lang.UnsupportedClassVersionError: com/android/tools/r8/D8 has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:756)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:473)
	at java.net.URLClassLoader.access$100(URLClassLoader.java:74)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:369)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:363)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:362)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
	at sun.launcher.LauncherHelper.checkAndLoadMain(LauncherHelper.java:621)
```